shell/common/gin_helper/event_emitter_caller.cc

```diff
@@ -4,6 +4,7 @@
 
 #include "shell/common/gin_helper/event_emitter_caller.h"
 
+#include "shell/browser/javascript_environment.h"
```

Comment:
Is this `include` needed?

Reply:
nope - i guess for a few of these we have no choice at present so i'll update them